sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1664,6 +1664,29 @@ def boundary_conditions(self):
         """
         return self._boundary_conditions
 
+    def polar_moment(self):
+        """
+        Returns the polar moment of area of the beam
+        about the X axis with respect to the centroid.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.beam import Beam3D
+        >>> from sympy import symbols
+        >>> l, E, G, I, A = symbols('l, E, G, I, A')
+        >>> b = Beam3D(l, E, G, I, A)
+        >>> b.polar_moment()
+        2*I
+        >>> I1 = [9, 15]
+        >>> b = Beam3D(l, E, G, I1, A)
+        >>> b.polar_moment()
+        24
+        """
+        if not hasattr(self.second_moment, '__iter__'):
```

Comment:
`from sympy.core.compatibility import iterable`
...
`if not iterable(self.second_moment):`